src/ui_components/search_input.js

```diff
@@ -45,7 +48,13 @@ export default class SearchInput {
     }
   }
 
+  store() {}
 
+  async restore(fragment) {
+    this.suggest.setValue(fragment)
+    let braggiResponse = await BragiPoi.get(fragment)
+    this.suggest.autocomplete.forceRender(braggiResponse)
```

Comment:
I don't think this can work.  
`items` are not updated, and no result can be selected.  
And what about results from favorites ?

I think we should try to stick to the existing suggest behavior.